.github/workflows/publish-to-pypi.yml

```diff
@@ -35,3 +34,5 @@ jobs:
       run: python -m build
     - name: Publish package
       uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        password: ${{ secrets.PYPI_API_TOKEN }}      
```

Comment:
This should change to a trusted publishing model, so no need for token: https://docs.pypi.org/trusted-publishers/